lib/config/rules/flowtype.js

```diff
@@ -13,6 +13,8 @@ module.exports = {
   'flowtype/no-dupe-keys': 'error',
   // Disallows use of primitive constructors as types, such as Boolean, Number and String.
   'flowtype/no-primitive-constructor-types': 'error',
+  // Disallows Flow type imports, aliases, and annotations in files missing a valid Flow file declaration (or a @noflow annotation). <FEEDBACK>
+  'flowtype/no-types-missing-file-annotation': 'off',
```

Comment:
Seems like this is something that should be turned on. See [rule docs](https://github.com/gajus/eslint-plugin-flowtype#eslint-plugin-flowtype-rules-no-types-missing-file-annotation) for more context. Thoughts?

@GoodForOneFare @lemonmade